<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spectrumSelection</class>
 <widget class="QDialog" name="spectrumSelection">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>169</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>spectrumSelection</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>File extension</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="pathLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Path: </string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="wUnitsCombobox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>um</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <item>
        <property name="text">
         <string>um</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>angstroms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nm</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLineEdit" name="fluxColumnLineEdit">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="placeholderText">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Flux units</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Flux column</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Wavelength column</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Wavelength units</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="redshiftLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="placeholderText">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="redshiftLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Redshift: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="extensionSelectCombobox">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Ascii file (.txt)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fits file (.fits)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="waveColumnLineEdit">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QComboBox" name="fUnitsCombobox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>erg/s/cm2/um</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mJy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>uJy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>W/m2/Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>erg/s/cm2/Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>W/m2/Angstroms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>erg/s/cm2/Angstroms</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QPushButton" name="fileButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QLabel" name="pathInputLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QPushButton" name="cancelButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="2">
      <widget class="QPushButton" name="acceptButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>spectrumSelection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>spectrumSelection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
